File: TwitterTest3.js
var apiManager = require("./ApiManagement.js");
var functionManager = require("./TwitterFunctions.js");

var apiManager = new apiManager();
var funcManager = new functionManager();

var mongo = require('mongodb');
var monk = require('monk');
var db = monk('localhost/data');
var htags = db.get('twitter');
// var list = db.get('list');

trendingHashtagArr = funcManager.requestTrendsAtLocation(23424977);
trendingHashtagArr.forEach(function(hashtag){
	// funcManager.addWordToColl(hashtag, list);
	arr[0] = hashtag;
	apiManager.processDataOnHashtags(arr, htags);
});
// cursor = list.find({"_id":1});
// str = JSON.parse(cursor[0]['list']);
// console.log(str);

File: TwitterTest3.py
__author__ = 'richardliu'

import requests
from requests_oauthlib import OAuth1
import json
from pymongo import MongoClient
from TwitterFunctions import *
from ApiManagement import *

if __name__ == "__main__":
    con = MongoClient()
    db = con.twitter
    list = db.list
    if list.count() == 0:
        list.insert({"_id":1, "list":''})
    trendingHashtagArr = requestTrendsAtLocation(23424977)
    for hashtag in trendingHashtagArr:
        addWordToColl(hashtag, list)

    cursor = list.find({"_id":1})
    str = json.dumps(cursor[0]['list'])
    print(str)
    con.close()

